uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -908,12 +900,12 @@ public function testManagerCanCancelOrRejectUserLeaveRequestEvenIfBalanceChangeI
     $updatedLeaveRequestData = $leaveRequestData;
     $updatedLeaveRequestData['id'] = $leaveRequest->id;
 
-    foreach (array("cancelled", "rejected") as $statusName) {
+    foreach (['cancelled', 'rejected'] as $statusName) {
```

Comment:
Instead of adding cancelled and rejected directly here. You can use `LeaveRequest::getCancelledStatuses`